src/edit.js

```diff
@@ -46,19 +55,32 @@ const edit = ({ attributes, className, setAttributes }) => {
 
 	const plainTextProps = {
 		value: attributes.content,
-		onChange: (content) => setAttributes({ content }),
-		placeholder: __('Write code…'),
-		'aria-label': __('Code'),
+		onChange: ( content ) => setAttributes( { content } ),
+		placeholder: __( 'Write code…' ),
+		'aria-label': __( 'Code' ),
 	};
 
+	const OldLightBlock = () =>
+		useLightBlockWrapper ? (
+			<Block.pre>
+				<PlainText
+					__experimentalVersion={ 2 }
+					tagName="code"
+					{ ...plainTextProps }
+				/>
+			</Block.pre>
+		) : (
+			<PlainText { ...plainTextProps } />
+		);
```

Comment:
On WordPress 5.5, there is a very annoying issue here where the cursor gets moved to the beginning after each change: https://youtu.be/yn9RdZQXZ4A

Reply:
Then, upon activating Gutenberg the block I had carefully created as:

![image](https://user-images.githubusercontent.com/134745/96517082-1e802c00-121d-11eb-874b-e454ad4a7c69.png)

Created gets rendered as this:

![image](https://user-images.githubusercontent.com/134745/96517015-04464e00-121d-11eb-8419-09f97eea07a2.png)

I think this is a problem introduced by https://github.com/WordPress/gutenberg/pull/24689. See https://github.com/WordPress/gutenberg/issues/26301.

Nevertheless, at least we when Gutenberg is active, I can enter text and not have the cursor jump back to the beginning.

Reply:
I was going back and testing against WP 5.3 and the behavior is the whole block loses focus. 
I'll look and test against the issues linked.

Reply:
I fixed this up, I found if I don't define the component in a variable and just use with an inline ternary then it works. I'm still not quite sure the difference here, but just committed a working version.

Reply:
Yeah, I found the same. I was likewise confused about why it would make any difference.